src/System.Security.Permissions/tests/project.json

```diff
@@ -0,0 +1,29 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.2-beta-devapi-24415-01",
```

Comment:
all of 24415-01 change to today's number 24418-01 

Dependency validation error: for System.Globalization 4.0.12-beta-devapi-24415-01 in /mnt/resource/j/workspace/dotnet_corefx/dev_api/centos7.1_debug_prtest/src/System.Security.Permissions/tests/project.json package prerelease is 'beta-devapi-24415-01', but expected 'beta-devapi-24418-01'